### ISSUE TITLE
Polarization and current density

### ISSUE THREAD

**Issue by gaurav123shukla** [ubermag/workshop#21]
![a nayak](https://user-images.githubusercontent.com/67150548/85912976-c43b0580-b7fe-11ea-9e8b-62859e0ff2a9.JPG)

Please describe briefly in next session, if we taken acount polarization P and current density j  i.e how to write command for this in in jupyter notebook. Sir please have a look ( highlighted text in attached jpef image) and suggest me how to write the comand lines in my jupyter notebook for this. My email id: gauravshukla571996@gmail.com. Thank you so much sir.

![a nayak 1](https://user-images.githubusercontent.com/67150548/85913037-6824b100-b7ff-11ea-99be-021764804a21.JPG)

**Comment by marijanbeg** [comment]
Hi @gaurav123shukla, thank you for your question. The dynamics term you need is Slonczewski (https://micromagneticmodel.readthedocs.io/en/latest/slonczewski.html). I am not sure we are going to have enough time to cover you question during the next session, but we might make a short tutorial about this topic.

**Comment by gaurav123shukla** [comment]
Yes sir, I request to you ,please make a short tutorial on this topic.
................................................
             Thanks and regards
           Gaurav Kumar Shukla
                Research Scholar
  School of Materials Science and Technology
                     IIT-VARANASI

**Comment by gaurav123shukla** [comment]
Sir please whenever suitable make a short tutorial (might be a pdf file) on above problem. Sir most of paper I see the give value of current density j value. Sir i don't know how to handle it. So if possible please help me.

**Comment by marijanbeg** [comment]
Could you please start composing a notebook about this, send it to me, and I am going to try to complete it for you?

**Comment by gaurav123shukla** [comment]
Sure sir!

**Comment by marijanbeg** [comment]
Great, when you finish the notebook, you can attach it in this issue. Please make sure you make a simple minimalistic example addressing the functionalities you need - no need to make a full paper reproducing notebook.

**Comment by gaurav123shukla** [comment]
Yes sir!

**Comment by gaurav123shukla** [comment]
Dear sir, I started with the initial parameters, now I am unable to find how nonodisk will form and apply current pulse only at that four positions. Please see it sir.


[antisyrmium.pdf](https://github.com/ubermag/workshop/files/4874939/antisyrmium.pdf)

**Comment by marijanbeg** [comment]
Thank you for the notebook. Could you please attach the `ipynb` file, so I can start from there.

**Comment by gaurav123shukla** [comment]
Thank you so much sir for comment.
Sir I am sending ipynb file.Please find attachment.
[Polarization and current density.zip](https://github.com/ubermag/workshop/files/4884529/Polarization.and.current.density.zip)

**Comment by gaurav123shukla** [comment]
Dear sir
Please attach any ipynb file including current density and polarization . I try to solve above posted problem myself.
